### ISSUE TITLE
MediaWiki 1.39 change to templatelinks table breaks the extension.

### ISSUE THREAD

**Issue by justinclloyd** [poke/mediawiki-ListTransclusions#4]
MediaWiki 1.39 throws the following error:

```
Error 1054: Unknown column 'tl_namespace' in 'field list'
Function: SpecialListTransclusions::loadTemplates
Query: SELECT tl_namespace,tl_title FROM `templatelinks` WHERE tl_from = 4092
```

The templatelinks table was changed for 1.39, dropping two fields as shown in the documentation and this Phabricator task, which also includes a comment on how to update queries that use the table:

https://www.mediawiki.org/wiki/Manual:Templatelinks_table
https://phabricator.wikimedia.org/T299417

**Comment by poke** [comment]
@justinclloyd Hey Justin,

sorry for the delays on this. I just published a fix for this with version 2.2. The extension is now verified to be working on both 1.39 and 1.41.

**Comment by justinclloyd** [comment]
Hi Patrick,

A test of this in the main dev wiki, on the main page, was resulting in errors due to the PHP process exceeding the 512 MB limit. I increased that for test purposes to 1 GB, which allowed the page to finish, but it ended up displaying 10 duplicate rows for each transcluded template, with seemingly hundreds of pages that aren't even templates in the results.

**Comment by poke** [comment]
@justinclloyd Hm, I’ll look into it. Can you tell me whether you have any custom configuration of [`$wgTemplateLinksSchemaMigrationStage`](https://www.mediawiki.org/wiki/Manual:$wgTemplateLinksSchemaMigrationStage)?

**Comment by justinclloyd** [comment]
I am not setting that variable, so it's using the default value.

**Comment by poke** [comment]
Okay, this should be fixed now for real with e4a40c6e1e.
